keps/sig-network/2943-networkpolicy-status/README.md

```diff
@@ -0,0 +1,462 @@
+
+### Version Skew Strategy
+
+N/A
+
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+-->
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: NetworkPolicyStatus
+  - Components depending on the feature gate:
+    - API Server
+    - Network Policy Providers
+
+
+###### Does enabling the feature change any default behavior?
+
+No
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+ Yes. Rolling back after Status has been added will keep 
+ the status persisted on ETCD but the field wont be accessible on the API Server for GET/PUT operations.
+
+Network Policy Providers wont be able to update the status field and 
+this may lead to this providers returning errors when trying to 
+access / change this field.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+Network Policy Providers will be able again to add the status conditions.
+###### Are there any tests for feature enablement/disablement?
+
+Not right now, but this can be targeted to Beta as soon as we have an initial implementation
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+A rollout won't cause any problem, as this is a new field.
+
+A rollback can be problematic from the Network Policy Provider perspective, 
+as if decided to use this field, the provider will have to deal with 
+failures if the Feature Gate is disabled or rolled back and the provider 
+already relies on this field to provide its status.
+
+###### What specific metrics should inform a rollback?
+
+  The increase of 5xx http error count on Network Policies Status subresource Endpoint
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+Should be targeted in Beta
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No
+
+### Monitoring Requirements
+
+
+###### How can an operator determine if the feature is in use by workloads?
+
+A new `network_policy_status` metric should be created and the amount 
+of operations executed on this endpoint should be monitored.
+
+
+###### How can someone using this feature know that it is working for their instance?
+
+- [X] API .status
+  - Condition name: All
+  - Other field: All
+
+The .status field will be created together with the Conditions
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+ - per-day percentage of API calls finishing with 5XX errors <= 1%
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+- [X] Metrics
+  - Metric name: network_policy_status
```

Comment:
We don't have such metric (and I'm assuming we won't add it) - I guess you meant the metric that I mentioned above?